mantidproject/mantid
# Default to old-style muon fitting UI

Rename "compatibility mode" to "Enable multiple fitting" and default it to off i.e. the old UI. The new fitting UI is still available by selecting the option.
(Request from scientists)

Although many files have been changed, this is mostly updating variable names - the actual change here is small. The [user documentation](https://github.com/mantidproject/mantid/blob/d5b16e1483814f877fe35a920ea61319c92ea83a/docs/source/interfaces/Muon_Analysis.rst#data-analysis), [developer documentation](https://github.com/mantidproject/mantid/blob/d5b16e1483814f877fe35a920ea61319c92ea83a/docs/source/interfaces/Muon_Analysis_developers_guide.rst#fitting-data) and [unscripted testing guide](https://github.com/mantidproject/mantid/blob/d5b16e1483814f877fe35a920ea61319c92ea83a/docs/source/interfaces/Muon_Analysis_unscripted_testing_guide.rst#setup) have all been updated.

If there is to be a patch for Mantid 3.8, the muon scientists would like this to be in it.
Otherwise, they will use the nightly build when this is merged in.

**To test:**

- Launch *Interfaces/Muon/Muon Analysis*
- Load any muon dataset (e.g. `MUSR00015190.nxs` from test data)
- Go to "Settings" tab. Check the option is called "Enable multiple fitting", that it is disabled by default, and that the tooltip makes sense.
- Go to "Data Analysis" tab. It should look like [this](http://docs.mantidproject.org/nightly/_images/MuonAnalysisDataAnalysis.png)
- Now go to the "Settings" tab and select the "Enable multiple fitting" option. The "Data Analysis" tab should now look like [this](http://docs.mantidproject.org/nightly/_images/MuonAnalysisDataAnalysis3.8.png)

Fixes #17972.

[Release notes](https://github.com/mantidproject/mantid/blob/d5b16e1483814f877fe35a920ea61319c92ea83a/docs/source/release/v3.9.0/muon.rst#id1) have been updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
